src/screens/explore-screen/components/ColorTile.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { ReactNode } from 'react'
+
+import {
+  Image,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+  ViewStyle
+} from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+import IconAudioRewardsPill from 'app/assets/images/iconAudioRewardsPill.svg'
+import Text from 'app/components/text/Text'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { font } from 'app/styles'
+import { ThemeColors } from 'app/utils/theme'
+
+type ColorTileProps = {
+  style?: ViewStyle
```

Comment:
```suggestion
  style?: StyleProp<ViewStyle>
```

This will allow an array of styles to be passed in too, like `style={[styles.tile, { opacity: 0.5 }]}`